scripts/crdb_backup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Take the current datetime:
+DT=`date +%Y-%m-%d`
+# Set the environment:
+set -o allexport
+source ../.env
+set +o allexport
+# Take the current datetime:
+DT=`date +%Y-%m-%d`
```

Comment:
why only add the day, it's probably better to include time as well (`.%H.%M.%S`)

Reply:
We want one backup per day for the moment and all servers will try to backup at different times. The first one will succeed, the rest will fail with "get an enterprise license".

Reply:
hm interesting, aren't we worried all nodes will fail and we won't notice?

Reply:
I'm only worried because a failed backup becomes part of the expected behaviour,
that's when mistakes or oversights happen 🤷‍♂️